### PR TITLE
Override log4j.properties when using provided

### PR DESCRIPTION
**Pull request by BJennWare**
log4j.properties can be provided for helm with --set-file FILE_PATH

### CODE REVIEW

File: charts/streams-app/templates/log4j-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+{{ if .Values.log4jConfig }}apiVersion: v1kind: ConfigMapmetadata:  name: {{ template "streams-app.name" . }}-log4j-configmapdata:  log4j.properties: |{{ .Values.log4jConfig | indent 4 }}{{ end }}
```
Comment (philipp94831) [inline charts/streams-app/templates/log4j-configmap.yaml]: Is that supposed to be just one line?

**Comment by philipp94831** [comment]
@BJennWare Please also make the changes to the statefulset

File: charts/streams-app/templates/log4j-configmap.yaml (hunk 1 above)
Comment (BJennWare) [reply charts/streams-app/templates/log4j-configmap.yaml]: This is strange. This file has 9 lines on my file system. I will push the file with linux line endings again

File: charts/streams-app/templates/log4j-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{{ if .Values.log4jConfig }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "streams-app.name" . }}-log4j-configmap
+data:
+  log4j.properties: |
+{{ .Values.log4jConfig | indent 4 }}
```
Comment (philipp94831) [inline charts/streams-app/templates/log4j-configmap.yaml]: What is the purpose of this? Can I still configure log4j.properties via values.yaml? Or just mount a file?

Comment (BJennWare) [reply charts/streams-app/templates/log4j-configmap.yaml]: We cannot just mount a file into helm. If we use helm --set-file log4jConfig=FILE_PATH than the contents of the file are in .Values.log4jConfig.

TLDR: We change the log config by mounting a log4j.properties file with --set-file

Comment (philipp94831) [reply charts/streams-app/templates/log4j-configmap.yaml]: Ok
